source/fiber_tasking_lib/global_args.h

```diff
@@ -15,6 +15,7 @@
 #include "fiber_tasking_lib/tagged_heap.h"
 #include "fiber_tasking_lib/tagged_heap_backed_linear_allocator.h"
 
+#define FTL_FIBER_STACK_SIZE 32768*4
```

Comment:
I think this would be more clear to be named:
FTL_HELPER_FIBER_STACK_SIZE

Since it's only used for helper fibers, not for the main fiber pool.